pkg/plugin/keyvault.go

```diff
@@ -115,6 +116,10 @@ func getVaultURL(vaultName string, azureEnvironment *azure.Environment) (vaultUR
 	if len(vaultName) < 3 || len(vaultName) > 24 {
 		return nil, fmt.Errorf("invalid vault name: %q, must be between 3 and 24 chars", vaultName)
 	}
+
+	//Trim quotes from vault name. https://github.com/Azure/kubernetes-kms/issues/85
+	vaultName = strings.Trim(vaultName, "\"")
```

Comment:
We'll need this for `keyName` and `keyVersion` too. Also, there could be a possibility of additional whitespace in the string. To account for that, we could probably define a `sanitize` function, that'll take a string, trim quotes, trim space and return the sanitized string.

Reply:
done